requirements.txt

```diff
@@ -4,8 +4,11 @@ lark-parser==0.7.8
 gunicorn==19.9.0
 flask-compress==1.4.0
 requests==2.23.0
+PyYAML==5.4
 attrs==19.3.0
 Flask-Commonmark==0.8
 bcrypt==3.2.0
 boto3==1.16.50
+levenshtein==0.12.0
 ruamel.yaml==0.17.4
+levenshtein==0.12.0
```

Comment:
Could we remove this extra line?

Reply:
It's necessary for the similar code finding implementation